File: 3790-fruits-into-baskets-ii/fruits-into-baskets-ii.cpp
class Solution {
public:
    int numOfUnplacedFruits(std::vector<int> &fruits, std::vector<int> &baskets) {
    int count = 0;

    for (int i = 0; i < fruits.size(); i++) {
        bool found = false;
        for (int j = 0; j < baskets.size(); j++) {
            if  (baskets[j] >= fruits[i]) {
                found = true;
                baskets[j] = INT_MIN;
                break;
            }
        }
        if (!found)
            count++;
    }
    return count;
}

};

File: 3790-fruits-into-baskets-ii/fruits-into-baskets-ii.kt
class Solution {
    fun numOfUnplacedFruits(fruits: IntArray, baskets: IntArray): Int {
    var count = 0
    for (i in 0..fruits.size - 1) {
        var found = false
        for (j in 0..baskets.size - 1){
            if (fruits[i] <= baskets[j]){
                found = true
                baskets[j] = Int.MIN_VALUE
                break
            }
        }
        if (!found)
            count++
    }
    return count
}
}